renovate.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@darahayes FYI - this will avoid many PR's and group all dependencies in single PR.
I think this is better than having 10 PR's to review on a daily basis as we not going to benefit with auto-merge and still need to review PR's. This option could save us some time on verifications.